easy_phi/tests/regression_test.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+import unittest
+import time
+
+import tornado.testing
+
+from easy_phi import app
+
+class Timer:
```

Comment:
Why not [timeit](https://docs.python.org/2/library/timeit.html)? It has better resolution